SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -808,9 +822,15 @@ void ActorSystem::AttachDynamicSubobject(AActor* Actor, Worker_EntityId EntityId
 			return;
 		}
 
+		if (Components.Find(ComponentId) == nullptr)
+		{
+			return;
+		}
+
 		ApplyComponentData(*Channel, *Subobject, ComponentId,
 						   SubView->GetView()[EntityId].Components.FindByPredicate(ComponentIdEquality{ ComponentId })->GetUnderlying());
-		PendingDynamicSubobjectComponents.Remove(MakeTuple(static_cast<Worker_EntityId_Key>(EntityId), ComponentId));
+
+		Components.Remove(ComponentId);
```

Comment:
Is the order important, or could this be RemoveSwap?

Reply:
This one's a TSet

Reply:
Code read fail by me.